test/integration/test_serialization.py

```diff
@@ -143,7 +142,6 @@ def test_qasm_job_result(self):
 
         self._verify_data(result.to_dict(), good_keys=good_keys)
 
-    @slow_test
     def test_pulse_job_result(self):
```

Comment:
Thought it would be easier to leave this test here because it uses private methods (`_verify_data`, `_find_potential_encoded`). It also runs pretty quickly locally - if it doesn't run properly in the integration suite I can move it to slow tests